### DISCUSSION TITLE
Is Ren'Py 7 supported?

### DISCUSSION THREAD

**Discussion by minding-myown-business** [renpy/vscode-language-renpy#436] [Q&A] [answered]
I noticed the name for the VSCode extension says 8.3+ at the end. All my projects use Ren'Py 7, so I was wondering if that is officially supported (or if it is not **officially** supported, is there a way to make it work?). Thanks!

**Comment by duckdoom4** [comment, marked as answer]
Hi, the extension will still work with the older renpy versions. Currently we don't really have a specific official version.
(This might change in the future).

The only thing that might not align correctly with older versions is highlighting appearing on keywords that don't exist in older versions (so new renpy 8+ keywords and features) and functions that are marked deprecated
